app/models/mixins/live_metrics_mixin.rb

```diff
@@ -0,0 +1,39 @@
+module LiveMetricsMixin
+  extend ActiveSupport::Concern
+
+  class MetricValidationError < RuntimeError; end
+
+  included do
+    def metrics_available
+      raise 'LiveMetricsMixing: metrics_available missing'
+    end
+
+    def metric(_metric, _start_time, _end_time, _interval)
+      raise 'LiveMetricsMixing: metric method missing'
+    end
+
+    def metrics(*metrics, start_time, end_time, interval)
```

Comment:
We don't usually use splats at the start of the method params.  I would expect this method to receive an Array.